benchmarking/driver/benchmark_driver.py

```diff
@@ -35,17 +35,19 @@ def runOneBenchmark(info, benchmark, framework, platform,
     if "shared_libs" in info:
         minfo["shared_libs"] = info["shared_libs"]
     try:
+        [1.0 for _ in range(20 << 20)]
```

Comment:
Nit: add the comment at the end of this line to explain the purpose (i.e., CPU cache invalidation.)